packages/oats/src/driver.ts

```diff
@@ -56,17 +63,22 @@ function defaultResolve() {
   return undefined;
 }
 
-export function localResolve(ref: string) {
+export function localResolve(ref: string, options: types.Options, kind?: NameKind) {
   if (ref[0] === '#') {
-    return { name: refToTypeName(ref) };
+    return {
+      name:
+        kind && options.nameMapper
```

Comment:
now this does not add the default prefixes if namemapper or kind is not given. Isn't that a mistake

Reply:
Kind of. If we don't do any manual `type` concatenation to resolved type name, a default prefix `type` is needed here.